src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -237,6 +238,10 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             (annotation: Annotation) => {
                 emitAnnotationUpdateEvent(annotation);
                 this.feedSuccessCallback();
+
```

Comment:
Instead of putting this logic in ActivitySidebar can we not toggle the replies in BaseComment in accordance with the resolution status of that Comment?

Reply:
Yeah we definitely could add a `useEffect` to BaseComment that would take care of this. Should I do that instead?

Reply:
That would be ideal if possible. We're doing a lot in ActivitySidebar already.

Reply:
I like this solution. I'm going to code this up in a separate branch

Reply:
Going to do this in a new branch since its a completely different solution